docs/index.rst

```diff
@@ -4120,6 +4119,52 @@ argument with code ``0`` to use ``last prediction`` and ``1`` for
     local_model.predict({"petal length": 3, "petal width": 1},
                         missing_strategy=PROPORTIONAL)
 
+If you would like a per-class prediction of the confidence or
```

Comment:
By the way, did you mention that this probability is the laplace corrected probability? And we should also metion that the "multiple" argument of the predict method will be deprecated